assets/training/finetune_acft_hf_nlp/environments/acpt-rft/context/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2204-cu126-py310-torch280:{{latest-image-tag:biweekly\.\d{6}\.\d{1}.*}}
+USER root
+RUN apt-get update && apt-get -y upgrade
+RUN pip install pip==25.3
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt --no-cache-dir
```

Comment:
have everything in requirements.txt, except maybe flash-attn 
latest conda from the base image should have pip 25.3

Reply:
1. The dependency graph don't allow that. for example, latest available xgrammer pulls older version of transformers. We lose required functionality if we allow it to install older transformers and requriements.txt will never succeed due to conflicting constraints on version. I've put only such necessary libraries in Docker file outside of requirements.txt